runtime/src/main/java/org/corfudb/runtime/view/AddressSpaceView.java

```diff
@@ -194,7 +207,7 @@ public void write(@Nonnull IToken token, @Nonnull Object data, @Nonnull CacheOpt
             } else {
                 ld = new LogData(DataType.DATA, data, runtime.getParameters().getCodecType());
             }
-            recordLogSizeDist(ld.getSizeEstimate());
+            recordLogSizeDist(sizeOf.deepSizeOf(ld));
```

Comment:
Just curious, have you measured how different this approach is to calling getSizeEstimate? Would be interesting to see the reported difference for the same kind of workload.

Reply:
Nope..I did not measure the time taken for getSizeEstimate. My guess would be that getSizeEstimate would be lower than deepSizeOf() as the latter one requires to walk through the object tree